Content.Shared/Interaction/SharedInteractionSystem.cs

```diff
@@ -292,12 +292,10 @@ protected bool ValidateInteractAndFace(EntityUid user, EntityCoordinates coordin
             Ignored? predicate = null,
             bool ignoreInsideBlocker = false)
         {
-            if (!origin.InRange(other, range)) return false;
-
             var dir = other.Position - origin.Position;
 
             if (dir.LengthSquared.Equals(0f)) return true;
-            if (range > 0f && !(dir.LengthSquared <= range * range)) return false;
+            if (range >= 0f && !(dir.LengthSquared < range * range)) return false;
```

Comment:
InRange also checks if the MapId is the same which this doesn't seem to do anymore.

I think you're better off replacing the range check here with the origin.InRange

Reply:
Fair enough

Reply:
I have replaced the check.